### PR TITLE
Add missing `npm install` step

### PR DESCRIPTION
**Pull request by krassowski**
## Reference Issues or PRs

Looking at https://github.com/nebari-dev/jhub-apps/pull/355 I noticed that dev install instructions did not include `npm install`

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ conda install -c conda-forge jhub-apps
 conda env create -f environment-dev.yml
 conda activate jhub-apps-dev
 pip install -e .
+cd ui
+npm install
```
Comment (jbouder) [inline README.md:42]: I would argue that this should actually be in Step 2 in the Developing Locally section. You should not need NPM dependencies installed to just run the app with the prebuilt assets. You just need those when you want to modify the ui part of the app.

That said, it doesn't hurt to add here, but maybe also add to the Developing Locally section

**Comment by jbouder** [comment]
@krassowski , just wanted to see if you need anything on this.

File: README.md (hunk 1 above)
Comment (krassowski) [reply README.md:42]: To me the Developing Locally section sounds like: "steps to run every time when developing locally after you already installed the dependencies" - as it does not install installation of other dependencies. That said, I see what you mean, what about:

~~~suggestion
```

To develop the React UI frontend, also run:

```bash
cd ui
npm install
cd -
```
~~~

Comment (jbouder) [reply README.md:42]: > To me the Developing Locally section sounds like: "steps to run every time when developing locally after you already installed the dependencies" - as it does not install installation of other dependencies. That said, I see what you mean, what about:

Works for me.

**Review by jbouder — APPROVED** [review]
LGTM

**Pull request merged**